packages/store-ui/src/atoms/Badge/Badge.tsx

```diff
@@ -9,7 +9,7 @@ export interface BadgeProps {
   children?: ReactNode
 }
 
-function Badge(
+const Badge = forwardRef<HTMLDivElement, BadgeProps>(function Badge(
   { testId = 'store-badge', children }: BadgeProps,
   ref: ForwardedRef<HTMLDivElement>
```

Comment:
if you use `forwardRef` directly on function, you don't need to use type to ref.

```suggestion
  ref
```